c++/src/kj/compat/http.h

```diff
@@ -643,6 +643,20 @@ class WebSocket {
 
   virtual uint64_t sentByteCount() = 0;
   virtual uint64_t receivedByteCount() = 0;
+
+  enum Role {
```

Comment:
I was between `Direction` and `Role`, went with the shorter one. Open to a better name!

Reply:
Hmm, at present it's not really clear which enum value I'm supposed to use for any particular context.

Let's change it to:

```
  enum ExtensionsContext {
    // Indicate whether a Sec-WebSocket-Extension header should be rendered for use in request
    // headers or response headers.
    REQUEST,
    RESPONSE
  };
```